scripts/veloxbench/veloxbench/_benchmark.py

```diff
@@ -0,0 +1,228 @@
+import datetime
+import functools
+import json
+import logging
+import os
+import subprocess
+from typing import Any, Dict, List, Optional, Tuple
+
+import conbench.runner
+
+logging.basicConfig(format="%(levelname)s: %(message)s")
+
+
+def _now_formatted() -> str:
+    now = datetime.datetime.now(datetime.timezone.utc)
+    return now.isoformat()
+
+
+def github_info(velox_info: Dict[str, Any]) -> Dict[str, Any]:
+    return {
+        "repository": "https://github.com/facebookincubator/velox",
+        "commit": velox_info["velox_git_revision"],
+    }
+
+
+def velox_info() -> Dict[str, Any]:
+    # TODO: undo the hard coding with ways to get the values from velox
+    # c.f. https://github.com/ursacomputing/benchmarks/blob/033eee0951adbf41931a2de95caccbac887da6ff/benchmarks/_benchmark.py#L30-L48
+    return {
+        "velox_version": "fake-version-0.0.1",
+        "velox_compiler_id": None,
+        "velox_compiler_version": None,
+        "velox_compiler_flags": None,
+        # This should work, though it would be much better to get the sha from velox directly so we know we're using the right one.
+        "velox_git_revision": subprocess.check_output(["git", "rev-parse", "HEAD"])
```

Comment:
For Arrow, we gather this from the arrow_info function in pyarrow that gives us nice info for these values. Is there a place we can programmatically get this from for velox?

Reply:
I think not yet. We're about to start working on pyvelox, so maybe we could create a similar api?

Reply:
Yeah, that's exactly what we do in Arrow (use pyarrow even in the cpp micro benchmarks) for this too.

In the mean time: we can use the git hash like this, are there other locations we might grep for that other info (like the cmake cache)? We have at least a compiler path + flags there. We don't need to do this right away though, we could punt on this.

Is there a velox version number somewhere we could grab?